recipes/azure-storage-cpp/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
```

Comment:
```suggestion

required_conan_version = ">=1.35.0"

```

The `apply_conandata_patches` requires 1.35.0